### PR TITLE
Update spawner UI config Images

### PR DESCRIPTION
**Pull request by jsitu777**
**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**
-Update images per v1.6.0 upstream

**Testing:**
- [ ] Unit tests pass
- [ ] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: awsconfigs/apps/jupyter-web-app/configs/spawner_ui_config.yaml [hunk 1]
```diff
@@ -17,10 +17,10 @@
 spawnerFormDefaults:
   image:
     # The container Image for the user's Jupyter Notebook
-    value: public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-scipy:v1.5.0
+    value: kubeflownotebookswg/jupyter-scipy:v1.6.0
```
Comment (surajkota) [inline awsconfigs/apps/jupyter-web-app/configs/spawner_ui_config.yaml:20]: lets change the default to DLC image `public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.6.3-cpu-py38-ubuntu20.04-v1.8`

**Review by surajkota — CHANGES_REQUESTED** [review]
nit comment

**Comment by surajkota** [comment]
Can you also update versions in following places?

KUBEFLOW_VERSION in tests/e2e/utils/constants.py
current development version in README.md
and 
https://github.com/awslabs/kubeflow-manifests/blob/main/website/content/en/docs/deployment/prerequisites.md#clone-the-repository

File: awsconfigs/apps/jupyter-web-app/configs/spawner_ui_config.yaml (hunk 1 above)
Comment (jsitu777) [reply awsconfigs/apps/jupyter-web-app/configs/spawner_ui_config.yaml]: changed.

**Review by AlexandreBrown — APPROVED** [review]

**Comment by jsitu777** [comment]
> Can you also update versions in following places?
> 
> KUBEFLOW_VERSION in tests/e2e/utils/constants.py current development version in README.md and https://github.com/awslabs/kubeflow-manifests/blob/main/website/content/en/docs/deployment/prerequisites.md#clone-the-repository

@surajkota all done

**Pull request closed without merging**